cloud/image_classification/README.md

```diff
@@ -71,7 +71,7 @@ cd ../../loadgen; CFLAGS="-std=c++14" python setup.py develop; cd ../cloud/image
 
 python setup.py develop
 ```
-
+Install the loadgen module under dist/ with pip install. Add -I to override any existing loadgen module..
```

Comment:
I think ```python setup.py develop``` is the best way of doing this.
The instructions in loadgen are not so good because they are for python2.7 where the benchmark app only works with python3.
Is there any specific issue why we want to change this?

Reply:
Sorry I have must have mixed the installation with the last set of instructions. It is all good.